external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -503,7 +495,7 @@ class AvroSuite extends QueryTest with SharedSQLContext with SQLTestUtils {
     // get the same values back.
     withTempPath { tempDir =>
       val name = "AvroTest"
-      val namespace = "org.apache.spark.avro"
+      val namespace = "com.databricks.spark.avro"
```

Comment:
why change the name space in test?

Reply:
Just revert the whole commit in #21878.
This line is from https://github.com/apache/spark/pull/21878/files#diff-9364b0610f92b3cc35a4bc43a80751bfL459